src/pages/credentials/useCredentials.tsx

```diff
@@ -1,11 +1,16 @@
 import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
 import IssuedList from 'src/modules/Credential/containers/IssuedList';
 
 export const useCredentials = () => {
   const { t: translate } = useTranslation();
+  const navigate = useNavigate();
   const tabs = [{ label: translate('credential-tab1'), content: <IssuedList /> }];
 
+  const onCreateCredential = () => navigate('create');
+
```

Comment:
this should be '../create'

Reply:
it's on `/credentials` route, and i wanna navigate to `/credentials/create` not `/create`. so it's ok